readyplayerme/rpm-unity-sdk-webview
# [SDK-414] feat: extract shared functionality

## [SDK-414](https://ready-player-me.atlassian.net/browse/SDK-414)

## Description

-   moved shared data and classes to Core package
- - this is linked to this PR for Core https://github.com/readyplayerme/rpm-unity-sdk-core/pull/125






## How to Test

-   Import or check out this branch, in combination with the feature branch that I made on Core `feature/extract-webview-logic`



## Checklist








[SDK-414]: https://ready-player-me.atlassian.net/browse/SDK-414?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
